x-pack/platform/plugins/shared/stack_connectors/public/connector_types/webhook/webhook_connectors.tsx

```diff
@@ -74,7 +77,13 @@ const WebhookActionConnectorFields: React.FunctionComponent<ActionConnectorField
         </EuiFlexItem>
       </EuiFlexGroup>
       <EuiSpacer size="m" />
-      <AuthConfig readOnly={readOnly} isPfxEnabled={isPfxEnabled} />
+      <React.Suspense fallback={<EuiLoadingSpinner size="m" />}>
```

Comment:
lazy loading the auth config. I did this initially because of bundle size, but I'm keeping it because it's fast even with slow 3g